src/Chainweb/Chainweb.hs

```diff
@@ -202,6 +203,7 @@ defaultChainwebConfiguration v = ChainwebConfiguration
     , _configIncludeOrigin = True
     , _configThrottleRate = 1000
     , _configMempoolP2p = defaultMempoolP2pConfig
+    , _configPruneChainDatabase = True
```

Comment:
`True` by default?

Reply:
yes, that's the current behavior. It's fast and it can save a lot of time in pact replay.